modules/efk-logging-elasticsearch-add-remove.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-elasticsearch.adoc
+
+[id='efk-logging-elasticsearch-add-remove_{context}']
+= Changing the number of Elasticsearch nodes
+
+You can scale the number of nodes in Elasticsearch.
+
+For example, if you want to increase redundancy, and use the `FulRedundancy` or `MultipleRedundancy` policy, you can add additional data nodes to increase the number of shard replicas in your cluster.  
+
+A {product-title} Elasticsearch cluster can contain three node types, or roles:
+
+* *Master*. A master node performs actions across the cluster, such as creating or deleting an index, tracking which nodes are part of the cluster, and deciding which shards to allocate to which nodes. The master node must have access to the the *_data/_* directory. You can create 1 or 3 master nodes.
+* *Data*. A data node is where {product-title} stores Elasticsearch shards.
+* *Client*. A client node routes requests, handles the Elasticsearch reduce phase, and distributes bulk indexing. 
+
+A node can have any combination of these roles.
+
+.Procedure
+
+. To add a node, edit the cluster logging Custom Resource Definition (CR), by default *_../hack/cr.yaml_* to add a number of modes of a specific type:
```

Comment:
I would remove reference to the 'hack/cr.yaml' as this is not necessarily representative of the default. The one in the CSV is probably more accurate if we need to provide anything: https://github.com/operator-framework/community-operators/blob/master/community-operators/cluster-logging/clusterlogging.v0.0.1.clusterserviceversion.yaml#L16

Reply:
@jcantrill What is the name of the cluster logging CRD that the user should edit? I reference editing the CRD elsewhere in other PRs. Is _clusterlogging.v0.0.1.clusterserviceversion.yaml_ at your link the CRD??

Reply:
They should never edit the CRD as this is the schema.  They will edit their CR which could be named anything they like.